lib/node-utils.js

```diff
@@ -693,7 +719,7 @@ function convertTokens(ast) {
      * @returns {undefined}
      */
     function walk(node) {
```

Comment:
I also thought about keeping somehow keeping track of if the parent node is a comment and passing it through the recursive `walk` function here (it could just be a boolean flag or options object). Now that I think about it, that might actually be the best way to achieve this since we probably don't really want to be potentially crawling up the tree on every node when we don't have to.

And again, if there's a better way to tell if a node is inside a comment from some value on the AST or some util we already have, please let me know!

Reply:
Realized there's an even simpler way to do this - please see later commits.